Amplify/Categories/DataStore/Model/Internal/Embedded.swift

```diff
@@ -9,6 +9,7 @@ import Foundation
 
 // MARK: - Embeddable
 
+/// Note that although this is public, it is intended for internal use and not consumed directly by host applications.
```

Comment:
Let's have something a little more noticeable and dire:

```swift
/// <rest of comment>
/// - Warning: Although this has `public` access, it is intended for internal use and should not be used directly
///   by host applications. The behavior of this may change without warning.
```

Reply:
thanks, taking this comment and applying it below the existing comment for classes. removing from extensions